noah/src/main/scala/edu/uci/ics/cloudberry/noah/crawler/CrawlerDriver.scala

```diff
@@ -0,0 +1,65 @@
+package edu.uci.ics.cloudberry.noah.crawler
+
+import java.io.File
+import java.util.concurrent.TimeUnit
+
+import com.crawljax.browser.EmbeddedBrowser.BrowserType
+import com.crawljax.condition.UrlCondition
+import com.crawljax.core.CrawljaxRunner
+import com.crawljax.core.configuration.CrawljaxConfiguration
+import com.crawljax.plugins.crawloverview.CrawlOverview
+import com.typesafe.config.ConfigFactory
+/**
+  * Created by ashka on 6/21/2016.
+  */
+object CrawlerDriver extends App{
+  val configPath = "C:\\Users\\ashka\\IdeaProjects\\cloudberry\\noah\\src\\main\\resources\\crawler\\promedmail.conf"
```

Comment:
Donot use absolute path which only work on local machines. 
To access the resource, you can follow this post: http://stackoverflow.com/questions/27360977/how-to-read-files-from-resources-folder-in-scala